hs-bindgen/hs-bindgen.cabal

```diff
@@ -250,12 +251,23 @@ executable clang-ast-dump
       -- External dependencies
     , optparse-applicative >= 0.18 && < 0.19
 
-test-suite golden
+test-suite tests
   import:         lang
   type:           exitcode-stdio-1.0
-  main-is:        golden.hs
+  main-is:        Main.hs
```

Comment:
My principle is to name `main-is` files reflecting componetn name, `hs-bindgen-tests.hs` (or previously `golden.hs`).

Because in big projects ,like this one, you'll end up with multiple `Main.hs` files, and it gets very confusing very fast.